docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md

```diff
@@ -81,39 +90,52 @@ class HomepageController extends Controller
 }
 ```
 
-This controller searches for all visible Content items of the type **Ride** (lines 37-38)
-and renders them using the `list/rides.html.twig` template (line 23-24).
+This controller searches for all visible Content items of the type **Ride** (lines 47-48)
+and renders them using the `list/rides.html.twig` template (line 33-34).
+
+### Register the controller
+
+!!! tip
+
+    Dealing with Symfony services is not in the scope of this tutorial, so we won't go here into detail on how they work.
+    To learn more about this topic, take a look at [Symfony Service Container documentation](https://symfony.com/doc/3.4/service_container.html).
```

Comment:
~~Dealing with Symfony services is not in the scope of this tutorial, so we won't go here into detail on how they work.~~
To learn how to use Symfony services, see [Symfony Service Container documentation](https://symfony.com/doc/3.4/service_container.html).